src/common/ServerAPI.jsx

```diff
@@ -16,27 +22,49 @@ async function createAccount(email, password) {
     masterKey
   );
 
-  return axios.post(
-    CREATE_ACCOUNT_URL,
-    {
-      email: email,
-      password: passwordHash,
-      encrypted_symmetric_key: encryptedSymetricKey,
-    },
-  );
+  return axios.post(CREATE_ACCOUNT_URL, {
+    email: email,
+    password: passwordHash,
+    encrypted_symmetric_key: encryptedSymetricKey,
+  });
 }
 
 async function loginAccount(email, password) {
   const masterKey = await deriveMasterKey(email, password);
   const passwordHash = await deriveMasterPasswordHash(password, masterKey);
 
-  return axios.post(
-    LOGIN_ACCOUNT_URL,
-    {
-      email: email,
-      password: passwordHash,
-    },
-  );
+  return axios.post(LOGIN_ACCOUNT_URL, {
+    email: email,
+    password: passwordHash,
+  });
+}
+
+async function getUserKey() {
+  return axios.get(USER_KEY);
+}
+
+async function getCollections() {
+  return axios.get(COLLECTIONS_URL);
+}
+
+async function postCollections(collectionName) {
+  return axios.post(COLLECTIONS_URL, {
+    name: collectionName,
+  });
+}
+
+async function postVaultItem(item, key, collectionUUID) {
+  return axios.post(COLLECTIONS_URL, {
```

Comment:
Sorry I keep seeing other things to ask about after I leave a review. 

Are we creating a vault item here? Should we use ITEMS_URL instead?